### ISSUE TITLE
No CMENU template

### ISSUE THREAD

**Issue by fizi** [e107inc/e107#2253]
The system only have the page_template.php. Not custom menu template. In this way I can't change the custom menu template. Link "<a>" immediately after text...I don't put it on the new line.

**Comment by CaMer0n** [comment]
menu_template.php

**Comment by fizi** [comment]
Sorry, I don't found it. You can close this.

**Comment by Moc** [comment]
I don't understand your answer. Did you find what you were looking for or do you need some more help? 
Could you specify which link you are referring to?

**Comment by fizi** [comment]
I found the template, please close this.

**Comment by rica-carv** [comment]
@fizi You can also close your own created issues, here in github...
